tools/mo/openvino/tools/mo/middle/passes/conv.py

```diff
@@ -48,9 +51,9 @@ def pad_op_transform(graph: Graph, match: dict):
     if op.type == 'Pooling':
         op['exclude_pad'] = False
     assert (graph[match['pad_output'].node][match['op'].node][0]['in'] == 0)
-    edge_attrs = graph.get_edge_data(match['pad_output'].id, match['op'].id)[0]
-    graph.remove_edge(match['pad_output'].id, match['op'].id)
-    graph.add_edge(input_data.id, match['op'].id, **{'in': 0, **edge_attrs})
+
+    match['op'].in_port(0).disconnect()
+    pad_op.in_port(0).get_connection().add_destination(match['op'].in_port(0))
```

Comment:
Is this correct? Won' we miss some data from edges here? @popovaan please comment